### ISSUE TITLE
Last subtitle line dont show up

### ISSUE THREAD

**Issue by EduFdezSoy** [jellyfin/jellyfin#4436]
The last line of the subtitles is not showing, only the last line. 

Tested in:
- Firefox: cuts the last line
- Android (exoplayer): working as expected
- VLC: working as expected  
  
The files tested are all **mkv** with **ass** subtitles inside  

I checked the subtitles but cant see anything wrong inside, here is the last line:  
`Dialogue: 0,0:10:25.58,0:10:29.17,Italics,Honda,0000,0000,0000,,That is the task of a comic bookstore.`  

---
**System:**
 - OS: Debian
 - Virtualization: none
 - Clients: Browser
 - Browser: Firefox 82.0.2 (64-bit)
 - Jellyfin Version: 10.6.4
 - Playback: DirectPlay  
 - Reverse Proxy: none
 - Base URL: none
 - Networking: host, local network under a private wiregard vpn 
 - Storage: local

**Comment by avi-grtl** [comment]
I'm having this same issue with .mkv and .ass subtitles dropping the last line of the subtitle from playback. Last line confirmed working in another player.

- Client OS: Windows 10
- Server OS: Unraid 6.8.3
- Jellyfin Version: 10.7.1 from Linuxserver.io docker container 
- Player: Firefox 86.0.1 (64-bit)
- Playback: Remux

Changing `Burn Subtitles` from `Auto` to `All Complex formats` works around this, but causes transcoding to occur when otherwise it wouldn't need to.

**Comment by SightSpirit** [comment]
Can confirm this problem on ExoPlayer, as well. Running 10.7.5 on Windows 10; client is official app 2.2.3 (Google Play build) running on Pixel 2 XL with Android 11. Affected subtitles are in VODSUB format.

**Comment by jjhhgg100123** [comment]
Also experiencing this on Jellyfin 10.7.7 running on Ubuntu. I also have issues with the subtitles disappearing if I skip in the middle of a line, that may be unrelated though.

Tested both direct play and transcoded, on Firefox 95.0b2, and Chrome 95.0.4638, on the latest W10.

File is H264, AAC, SSA subtitles. Plays normally in VLC outside of Jellyfin.

**Comment by Shados** [comment]
Yeah, I'm getting this issue as well. The last subtitle shows up fine if I use mpv or vlc to play the file directly, but Jellyfin fails to display it.
